egs2/msropenin/asr1/Fusing_ASR_frontends_in_Espnet2.md

```diff
@@ -0,0 +1,74 @@
+# Fusing ASR frontends in EspNET 2 
```

Comment:
It may not be needed.
If this description is important to be kept, leave it as it is.
Otherwise, you can remove this.

Reply:
Removed this